compose/service.py

```diff
@@ -787,22 +786,22 @@ def pull(self, ignore_pull_failures=False):
                 log.error(six.text_type(e))
 
 
-class Net(object):
```

Comment:
We are still using `Net`, `ContainerNet` and `ServiceNet` in `tests/integration/service_test.py` and `tests/unit/service_test.py`, causing failures.